src/draw/arm2d/lv_gpu_arm2d.c

```diff
@@ -84,12 +87,12 @@
 #define __arm_2d_impl_colour_filling_mask_opacity                               \
     __arm_2d_impl_rgb565_colour_filling_mask_opacity
 #define __arm_2d_impl_copy              __arm_2d_impl_rgb16_copy
-#define __arm_2d_impl_alpha_blending    __arm_2d_impl_rgb565_alpha_blending
+#define __arm_2d_impl_tile_copy_opacity    __arm_2d_impl_rgb565_tile_copy_opacity
 #define __arm_2d_impl_src_msk_copy      __arm_2d_impl_rgb565_src_msk_copy
 #define __arm_2d_impl_src_chn_msk_copy  __arm_2d_impl_rgb565_src_chn_msk_copy
 #define __arm_2d_impl_cl_key_copy       __arm_2d_impl_rgb16_cl_key_copy
-#define __arm_2d_impl_alpha_blending_colour_keying                              \
-    __arm_2d_impl_rgb565_alpha_blending_colour_keying
+#define __arm_2d_impl_tile_copy_colour_keying_opacity                           \
```

Comment:
FYI, on other places in LVGL we use the US "color" spelling. As these defines are not part of the public API we can stick with yours/ARMs flavor.